mumax3c/tests/conftest.py

```diff
@@ -2,13 +2,81 @@
 
 import mumax3c as mc
 
+not_supported_by_mumax = [
+    "TestDemag.test_demag_asymptotic_radius",
+    "TestEnergy.test_zeeman_zeeman",
+    "TestExchange.test_field",
+    "TestHysteresisDriver.test_noevolver_nodriver",
+    "TestMinDriver.test_evolver_nodriver",
+    "TestMinDriver.test_evolver_driver",
+    "TestMinDriver.test_wrong_evolver",
+    "test_outputstep",
+    "TestPrecession.test_dict",
+    "TestPrecession.test_field",
+    "TestRKKY.test_scalar",
+    "TestThreads.test_threads",
+    "TestTimeDriver.test_rungekutta_evolver_nodriver",
+    "TestTimeDriver.test_euler_evolver_nodriver",
+    "TestTimeDriver.test_theta_evolver_nodriver",
+    "TestTimeDriver.test_therm_heun_evolver_nodriver",
+    "TestTimeDriver.test_noevolver_nodriver_finite_temperature",
+    "TestTimeDriver.test_wrong_evolver",
+    "TestTimeDriver.test_noevolver_driver",
+    "TestZeeman.test_time_vector",
+    "TestZeeman.test_time_dict",
+    "TestZeeman.test_time_field",
+    "TestMinDriver.test_noevolver_driver",
+    "TestDMI.test_crystalclass",
```

Comment:
If this test fails due to missing crystalclasses we sholud split the test method.